FunToolbox/Prelude.fs

```diff
@@ -4,7 +4,6 @@ open System
 open System.Threading
 open System.Threading.Tasks
 
-type CRAttribute = CompilationRepresentationAttribute

```

Comment:
The `CRAttribute` type alias should not be removed because dependent projects - even if they only support F#4.1 - may need it if a module augments a type in a different source file.